### DISCUSSION TITLE
cant deploy anymore

### DISCUSSION THREAD

**Discussion by d-opri** [neuefische/questions#534] [Web] [answered]
hey hey,
i am struggling with deploying my app after making some changes. it first gave me an error regarding the import link for my font so i added this rule to my eslintrc
`
rules": { "react/no-unescaped-entities": 0 }`

but now i receive several other errors, i think it makes more sense to link the whole [status from vercel](https://vercel.com/d-opri/kroko-capstone/CnP5AjUFL6FDsGRCuLHnWTPU8qUM)

i tried redeploying but that also didnt work.
any help is appreciated!

**Comment by shebtastic** [comment, marked as answer]
hey!
we can't view the logs, because the can only be viewed by collaborators on the vercel project.
it would be a great help, if you could copy paste them here though. 😁

how did you import your font? did you use [`next/font`](https://nextjs.org/docs/basic-features/font-optimization)? a link to code would help a lot here.

your linting error should not be the reason why the project fails to deploy. as such the linting rule you added will only silence the check, not avoid the code style issues that ESLint found.

**Reply by d-opri** [reply]
i see, thanks for the tip. here the logs
<img width="1046" alt="Screenshot 2023-02-15 at 14 35 57" src="https://user-images.githubusercontent.com/118123298/219042509-35e39b79-f5a3-46dd-864c-4eb0912eb865.png">
<img width="1151" alt="Screenshot 2023-02-15 at 14 35 52" src="https://user-images.githubusercontent.com/118123298/219042516-fe379312-9c6c-4d3b-b2e1-49ad726cd841.png">

initially i had just Typography without the .js since i dont need it for the otehrs but i wanted to try and see if it was that

thannk!!

**Reply by mbosselmann** [reply]
Hi Danie, if I saw it correctly the name of your file is "typography.js" and not "Typography.js". This may be the problem. 😅

**Reply by d-opri** [reply]
hey! i had the file renamed since its in my components folder, so now its Typography. do you mean i have it soomewhere in lower case?
